lib/broc-component-css-preprocessor.js

```diff
@@ -92,7 +93,21 @@ BrocComponentCssPreprocessor.prototype.write = function (readTree, destDir) {
         var podGuid = podPath.join('--') + '-' + guid();
         var fileContents = fs.readFileSync(path.join(srcDir, filepath)).toString();
 
-        buffer.push(processors[pod.extension](fileContents, podGuid));
+        if(pod.extension === 'scss') {
+          var outFilepath = filepath;
+          // TODO: introduce a useComponentNameAsFileName option which defaults to false
+          if (true) {
```

Comment:
You can set this in the constructor. Probably as `this.options.useComponentNameAsFileName`?